server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java

```diff
@@ -117,13 +122,15 @@ public void upgradeRoot(@NonNull ServerContext context) {
     log.debug("Upgrade root: upgrading to data version {}", METADATA_FILE_JSON_ENCODING);
     var rootName = Ample.DataLevel.METADATA.metaTable();
     upgradeTabletsMetadata(context, rootName);
+    removeScanServerRange(context, Ample.DataLevel.METADATA.metaTable());
```

Comment:
It may be that this code is not needed because looking back at the 2.1 code it may always put scan server refs in the metadata table and never the root table.  However it does not hurt to run this if there is nothing to delete then it will do nothing.  I was looking into this to see if there was anything to delete from ZK and there does not seem to be.

Unrelated to this PR it may be that when scanning the metadata table in 2.1 w/ scans servers that it puts the refs in the metadata table itself.  Or even when scanning the root table w/ scan servers it may put the refs in the metadata table.

Reply:
Yeah I wasn't sure if scan server refs ended up in the root table or not. 

When trying to write test code for the upgrader, I was hitting a lot of metadata constraint violations due to the refs not being allowed. 

I can manually test this locally and see if the refs are ever written to the root table.

Reply:
Was able to work around the constraint issue by removing the constraints as the system user before writing the old scan refs. 

Still need to test the root scan ref stuff locally.

Reply:
Tested scanning the root table with scan servers and found that scanrefs were only ever added to the metadata table. 

```
root@uno> scan -t accumulo.metadata --consistency-level eventual
+rep< srv:dir []	default_tablet
+rep< srv:time []	L0
+rep< ~tab:~pr []	\x00
root@uno> scan -t accumulo.metadata 
+rep< srv:dir []	default_tablet
+rep< srv:time []	L0
+rep< ~tab:~pr []	\x00
~sservhdfs://localhost:8020/accumulo/tables/!0/table_info/0_1.rf localhost:9996:4182bfc1-9f7a-4852-8011-fe04aac71961 []
~sservhdfs://localhost:8020/accumulo/tables/+r/root_tablet/A000000p.rf localhost:9996:4182bfc1-9f7a-4852-8011-fe04aac71961 []
```

I removed the upgrade steps for the root table